lib/inferno/dsl/fhir_validation.rb

```diff
@@ -151,13 +151,15 @@ def issue_severity(issue)
         end
 
         # @private
-        def issue_message(issue)
+        def issue_message(issue, resource)
           location = if issue.respond_to?(:expression)
                        issue.expression&.join(', ')
                      else
                        issue.location&.join(', ')
                      end
 
+          location.prepend("#{resource.resourceType}#{"/#{resource.id}" if resource.id}/")
```

Comment:
```suggestion
          location.prepend("#{resource.resourceType}#{"/#{resource.id}" if resource.id}: ")
```

Ifs inside string interpolations are kind of gross, so how about something like this instead:
```
location_prefix = resource.id ? "#{resource.resourceType}/#{resource.id}" : resource.resourceType
```